### PR TITLE
flatpak: add desktop and metainfo

### PR DESCRIPTION
**Pull request by shumvgolove**

### CODE REVIEW

File: scripts/flatpak/chat.simplex.simplex.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  </developer>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>AGPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>SimpleX Chat - a private and encrypted messenger without any user IDs (not even random ones)! Security assessment was done by Trail of Bits in November 2022.</p>      
+    <p>SimpleX Chat features:</p>
+    <ol>
+      <li>end-to-end encrypted messages, with editing, replies and deletion of messages.</li>
```
Comment (spaced4ndy) [inline scripts/flatpak/chat.simplex.simplex.metainfo.xml:19]: seems dated compared to stores

File: scripts/flatpak/chat.simplex.simplex.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Terminal=false
+Name=SimpleX Chat
+Comment=A private and encrypted messenger without any user ID
```
Comment (epoberezkin) [inline scripts/flatpak/chat.simplex.simplex.desktop:5]:
```suggestion
Comment=A private & encrypted open-source messenger without any user IDs (not even random ones)!
```

File: scripts/flatpak/chat.simplex.simplex.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>chat.simplex.simplex</id>
+  
+  <name>SimpleX Chat</name>
+  <summary>SimpleX - the first messaging platform that has no user identifiers, not even random numbers!</summary>
```
Comment (epoberezkin) [inline scripts/flatpak/chat.simplex.simplex.metainfo.xml:6]:
```suggestion
  <summary>SimpleX - a private & encrypted open-source messenger without any user IDs (not even random ones)!</summary>
```

File: scripts/flatpak/chat.simplex.simplex.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>chat.simplex.simplex</id>
+  
+  <name>SimpleX Chat</name>
+  <summary>SimpleX - a private & encrypted open-source messenger without any user IDs (not even random ones)!</summary>
```
Comment (epoberezkin) [inline scripts/flatpak/chat.simplex.simplex.metainfo.xml:6]:
```suggestion
  <summary>SimpleX - a private &amp; encrypted open-source messenger without any user IDs (not even random ones)!</summary>
```

File: scripts/flatpak/chat.simplex.simplex.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>chat.simplex.simplex</id>
+  
+  <name>SimpleX Chat</name>
+  <summary>SimpleX - a private &amp; encrypted open-source messenger without any user IDs (not even random ones)!</summary>
```
Comment (epoberezkin) [inline scripts/flatpak/chat.simplex.simplex.metainfo.xml:6]:
```suggestion
  <summary>SimpleX - a private by design open-source messenger without any user IDs (not even random)!</summary>
```

Comment (epoberezkin) [inline scripts/flatpak/chat.simplex.simplex.metainfo.xml:6]:
```suggestion
  <summary>A private &amp; encrypted open-source messenger without any user IDs (not even random)!</summary>
```

**Pull request merged**